libs/build_met.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash                                                                                                                              
+
+set -eux
+
+name="met"
+version=${1:-${STACK_met_version}}
+release_date=${2:-${STACK_met_release_date}}
+
+cd  ${HPC_STACK_ROOT}/${PKGDIR:-"pkg"}
+software=$name-$version.$release_date
+pkg_name=$name-$version
+url="https://github.com/dtcenter/MET/releases/download/v$version/$software.tar.gz"
```

Comment:
If this is available on github, can you please use git clone instead of wget?
Many platforms allow git clone but disable wget operations.

Reply:
@aerorahul 
Some portion of the released MET code is auto-generated when a release is created. We run a script (make dist) to create a release tarfile that generates the code and also strips out unnecessary code for development utilities from the release tarfile. Compiling the code directly from the repository would require a couple of additional dependencies. For these reasons, it's simpler to use the released tarfile instead of compiling directly from the repository. If doing so is a requirement, we could consider revising some of the code infrastructure we've developed to accommodate that request.

This build_met.sh script has only been tested on and for WCOSS (and no other machines) using the  config_wcoss_dell_p3.sh file in order to expedite the process of getting this into hpc-stack.  It was my understanding that once I got this working and merged, I would add to the other config files for other machines and test there and add those to the repositories, but I can do that for the other machines as time allows before this pull request is approved if you would prefer.